core/net/rpl/rpl.c

```diff
@@ -139,7 +139,7 @@ rpl_purge_routes(void)
       PRINTF("No more routes to ");
       PRINT6ADDR(&prefix);
       dag = default_instance->current_dag;
-      /* Propagate this information with a No-Path DAO to preferred parent if we are not a RPL Root */
+      /* Propagate this information with a No-Path DAO to preferred parent if we are not an RPL Root */
```

Comment:
With all due respect, how would a reader know that!?

On January 4, 2015 6:30:16 PM EST, Adam Dunkels <notifications@github.com> wrote:
>> @@ -139,7 +139,7 @@ rpl_purge_routes(void)
>>        PRINTF("No more routes to ");
>>        PRINT6ADDR(&prefix);
>>        dag = default_instance->current_dag;
>> -      /* Propagate this information with a No-Path DAO to preferred
>parent if we are not a RPL Root */
>> +      /* Propagate this information with a No-Path DAO to preferred
>parent if we are not an RPL Root */
>
>RPL is supposed to be pronounced "ripple", so it should be "a RPL root"
>and not "an RPL root":
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/contiki-os/contiki/pull/936/files#r22443975

-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.